### PR TITLE
Close streams in plugin_controller_test.dart

### PR DESCRIPTION
**Pull request by miguelcmedeiros**

### CODE REVIEW

**Comment by miguelcmedeiros** [comment]
wow, some stuff snuck in (formatting and configuration upgrade)... I'll fix it

File: test/plugin_controller_test.dart [hunk 1]
```diff
@@ -46,6 +46,13 @@ void main() {
       );
     });
 
+    tearDown(() {
+      _connectedDeviceStreamController.close();
+      _argsStreamController.close();
+      _scanStreamController.close();
+      _statusStreamController.close();
+    });
```
Comment (werediver) [inline test/plugin_controller_test.dart:54]: 👍

**Review by werediver — APPROVED** [review]
Thanks @miguelcmedeiros!

**Comment by miguelcmedeiros** [comment]
Formatting is now back to what it should be (line length of 80 characters).
About the configuration, it's actually OK to update to the latest, right?

**Pull request merged**